src/main/resources/static/js/patients.js

```
@@ -0,0 +1,284 @@
+    patientCards.forEach(card => {
+        let matchFound = false;
+
+        // Check patient name
+        const patientNameEl = card.querySelector('.patient-name');
+        if (patientNameEl && patientNameEl.textContent.toLowerCase().includes(searchTerm)) {
+            matchFound = true;
+        }
+
+        // Check patient ID
+        const patientIdEl = card.querySelector('.patient-id');
+        if (patientIdEl && patientIdEl.textContent.toLowerCase().includes(searchTerm)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)